3party/pugixml/utils.hpp

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "pugixml/src/pugixml.hpp"
+#include <pugixml.hpp>
```

Comment:
This change currently breaks iOS build, as the iOS build doesn't use CMake, and does not have the `pugixml/src/` among the include paths.
That should be fixed. If you need help with that then let me know.

Reply:
I have no clue about iOS (and don't really care personally) so yes I'd like some help there 🤔

Reply:
OK, I have added changes to get the iOS build compile,
in [this commit](https://github.com/Ferenc-/organicmaps/commit/2d2fffd2c6f0530bbdbfdbee6c6f57632c90a5a8) based on your current PR branch.  
You can `cherry-pick` that hash or `rebase` onto my [support-system-pugixml](https://github.com/Ferenc-/organicmaps/tree/support-system-pugixml) branch.